src/apps/startups/models.py

```diff
@@ -0,0 +1,47 @@
+import uuid
+
+from django.core.validators import RegexValidator
+from django.db import models
+from django.utils import timezone
+
+from apps.users.models import User, Founder
+
+
+# from pictures.models import PictureField
+
+
+class Startup(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    company_name = models.CharField("Company Name", max_length=64)
+
+    founders = models.ManyToManyField(Founder)
+
+    STARTUP_SIZE = (("S", "Small"), ("M", "Medium"), ("B", "Big"), ("L", "Large"))
+
+    size = models.CharField(
+        max_length=1,
+        choices=STARTUP_SIZE,
+        blank=True,
+        default="S",
+        help_text="Startup size",
+    )
+
+    phone_number = models.CharField(
+        max_length=16,
+        blank=False,
+        null=False,
+        validators=[
+            RegexValidator(
+                regex=r"^[\+]?[(]?[0-9]{3}[)]?[\s\.]?[0-9]{3}[\s\.]?[0-9]{4,6}$",
+                message="Phone number must be entered in the format '+123456789'. Up to 15 digits allowed.",
+            ),
+        ],
+    )
+
+    location = models.CharField(max_length=255, null=True)
```

Comment:
Allowing ```null=True``` for a ```CharField``` can lead to having two possible values for "no data": NULL and the empty string. It's usually recommended to use ```blank=True``` instead of ```null=True``` for string-based fields to avoid this ambiguity

Reply:
Fixed in 00be11774de59a7e0960628bb8455c8bfa06548d